ci/taos/plugins-staging/pr-nnstreamer-ubuntu-apptest.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+##
+# Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+##
+# @file pr-nnstreamer-ubuntu-apptest.sh
+# @brief Check nnstreamer sample app working well.
+# @see      https://github.com/nnsuite/TAOS-CI
+# @see      https://github.com/nnsuite/nnstreamer/wiki/usage-examples-screenshots
+# @author   Sewon Oh <sewon.oh@samsung.com>
+
+
+# @brief [MODULE] TAOS/pr-nnstreamer-ubuntu-apptest-wait-queue
+function pr-nnstreamer-ubuntu-apptest-wait-queue(){
+    message="Trigger: queued. There are other build jobs and we need to wait.. The commit number is $input_commit."
+    cibot_pr_report $TOKEN "pending" "TAOS/pr-nnstreamer-apptest" "$message" "${CISERVER}${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+}
+
+# @brief [MODULE] TAOS/pr-nnstreamer-ubuntu-apptest-ready-queue
+function pr-nnstreamer-ubuntu-apptest-ready-queue(){
+    echo "[DEBUG] Starting CI trigger to run nnstreamer sample app actually."
+    message="Trigger: running. The commit number is $input_commit."
+    cibot_pr_report $TOKEN "pending" "TAOS/pr-nnstreamer-apptest" "$message" "${CISERVER}${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+}
+
+# @brief [MODULE] TAOS/pr-nnstreamer-ubuntu-apptest-run-queue
+function pr-nnstreamer-ubuntu-apptest-run-queue() {
+    echo "########################################################################################"
+    echo "[MODULE] TAOS/pr-nnstreamer-apptest: Starting sample app test"
+    check_dependency cmake
+    check_dependency make
+    check_dependency python
```

Comment:
check_dependency wget

Please append the statement above (for strict check even though other modules checks wget.)